service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionServiceTest.java

```diff
@@ -55,7 +60,67 @@ void shouldConcatenateAllChildrenNames() {
             .build();
 
         Map<String, String> templateData = caseDataExtractionService.getNoticeOfProceedingTemplateData(caseData);
-        assertThat(templateData.get("childrenNames")).isEqualTo("Bran Stark, Sansa Stark");
+        assertThat(templateData.get("childrenNames")).isEqualTo("Bran Stark, Sansa Stark and Jon Snow");
+    }
+
+    @Test
+    void shouldNotApplySentenceFormattingToSingularChildName() {
+        CaseData caseData = CaseData.builder()
```

Comment:
can we extract some common stuff to a private method that returns CaseData.CaseDataBuilder? Then you can add the custom bits you need and shorten these big bits of set up

Reply:
Done so Sam, good shout